examples/cucumber/.sauce/config.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1alpha
+kind: espresso
+sauce:
+  region: us-west-1
+  concurrency: 10 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Cucumber and Espresso Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+espresso:
+  app: ./apps/espresso-cucumber.apk
+  testApp: ./apps/espresso-cucumber-success.apk
+suites:
+  - name: "saucy barista"
```

Comment:
@valentyn88 can we rename the suite name to `saucy cucumber espresso` ?

Reply:
sure